code/modules/antagonists/nations/factions/cargo.dm

```diff
@@ -0,0 +1,19 @@
+/datum/team/nationcargo
+	name = "Cargonia"
+	member_name = "Cargonians"
+
+/datum/antagonist/cargonia
+	name = "cargonia"
+	show_in_antagpanel = TRUE
+	antagpanel_category = "Nations"
+	job_rank = ROLE_NATIONS
+	show_name_in_check_antagonists = TRUE
+	var/datum/team/nation/nationcargo
+	var/hud_type = "nation"
+
+/datum/antagonist/cargonia/on_gain()
+	owner.special_role = ROLE_NATIONS
```

Comment:
This should go into antagonist/nations/on_gain() and cargonia, along with all the others, should be subtypes of antagonist/nations

Reply:
im dumb and was more concerned at the time of getting this at barebones working level, now that ive actually got the gamemode to assign antags properly I can start working on features